tests/v2_tests/test_score_modifier_search.py

```diff
@@ -77,11 +77,11 @@ def test_invalid_score_modifiers_format(self):
                 "multiply_score_bys":
                     [{"field_name": "multiply_1",
                       "weight": 1,},
-                     {"field_name": "multiply_2",}],
+                     {"field_name": "multiply_2.a",}],
                 "add_to_score": [
                     {"field_name": "add_1", "weight" : 4,
                      },
-                    {"field_name": "add_2", "weight": 1,
+                    {"field_name": "add_2.a", "weight": 1,
                      }]
             }
 
```

Comment:
I think the assertion for this test is too wide. we should at least test the error code and some key word in the response if there's any